src/workerd/api/node/tests/util-nodejs-test.js

```diff
@@ -25,24 +25,73 @@
 
 import assert from 'node:assert';
 import { mock } from 'node:test';
-import util, { inspect } from 'node:util';
+import util, {
+  inspect,
+  callbackify,
+  inherits,
+  promisify,
+  getCallSites,
+  emitExperimentalWarning,
+  stripVTControlCharacters,
+  styleText,
+} from 'node:util';
 
 const remainingMustCallErrors = new Set();
 function commonMustCall(f) {
   const error = new Error('Expected function to be called');
   remainingMustCallErrors.add(error);
-  return (...args) => {
+  return function (...args) {
     remainingMustCallErrors.delete(error);
-    return f(...args);
+    return f.call(this, ...args);
   };
 }
+function commonMustSucceed(fn, exact) {
```

Comment:
If we're going to implement these methods, it should be a joint effort. I recommend following the existing test syntax/methodologies.

Reply:
I'm happy to also do a more involved translation, I'm just thinking maybe it's easier to port and maintain if we do maintain some of these. And this file was already implementing them so I thought that pattern was okay.